nipype/interfaces/workbench/cifti.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""This module provides interfaces for workbench CIFTI commands"""
+from __future__ import (print_function, division, unicode_literals,
+                        absolute_import)
+
+from ..base import (TraitedSpec, File, traits, CommandLineInputSpec)
+from .base import WBCommand
+from ... import logging
+
+iflogger = logging.getLogger('nipype.interface')
+
+
+class CiftiSmoothInputSpec(CommandLineInputSpec):
+    in_file = File(
+        exists=True,
+        mandatory=True,
+        argstr="%s",
+        position=0,
+        desc="The input CIFTI file")
+    sigma_surf = traits.Float(
+        mandatory=True,
+        argstr="%s",
+        position=1,
+        desc="the sigma for the gaussian surface smoothing kernel, in mm")
+    sigma_vol = traits.Float(
+        mandatory=True,
+        argstr="%s",
+        position=2,
+        desc="the sigma for the gaussian volume smoothing kernel, in mm")
+    direction = traits.Enum(
+        "ROW",
+        "COLUMN",
+        mandatory=True,
+        argstr="%s",
+        position=3,
+        desc="which dimension to smooth along, ROW or COLUMN")
+    out_file = File(
+        name_source=["in_file"],
+        name_template="smoothed_%s.nii",
+        keep_extension=True,
+        argstr="%s",
+        position=4,
+        desc="The output CIFTI")
+    left_surf = File(
+        exists=True,
+        mandatory=True,
+        position=5,
+        argstr="-left-surface %s",
+        desc="Specify the left surface to use")
+    left_corrected_areas = File(
+        exists=True,
+        position=6,
```

Comment:
Are all these flagged arguments truly positional?

Reply:
i think not, but was staying true to how it appears to be written up
https://github.com/Washington-University/workbench/blob/683ce2e9658ad55a926f0dda947d4537c0360d78/src/Algorithms/AlgorithmCiftiSmoothing.cxx#L54